core/arch/arm/plat-totalcompute/platform_config.h

```diff
@@ -11,7 +11,9 @@
 /* Make stacks aligned to data cache line length */
 #define STACK_ALIGNMENT		64
 
-#if defined(PLATFORM_FLAVOR_tc0) || defined(PLATFORM_FLAVOR_tc1)
+#if (defined(PLATFORM_FLAVOR_tc0) ||	\
+defined(PLATFORM_FLAVOR_tc1) ||	\
+defined(PLATFORM_FLAVOR_tc2))
```

Comment:
fix indentation:
```c
#if (defined(PLATFORM_FLAVOR_tc0) ||	\
     defined(PLATFORM_FLAVOR_tc1) ||	\
     defined(PLATFORM_FLAVOR_tc2))

Reply:
>> checkpatch 8f64138f5480768c421b461a2053e146a940fe71
8f64138f plat-totalcompute: Introduce TC2
WARNING: please, no spaces at the start of a line
#43: FILE: core/arch/arm/plat-totalcompute/platform_config.h:15:
+     defined(PLATFORM_FLAVOR_tc1) ||^I\$

WARNING: please, no spaces at the start of a line
#44: FILE: core/arch/arm/plat-totalcompute/platform_config.h:16:
+     defined(PLATFORM_FLAVOR_tc2))$

total: 0 errors, 2 warnings, 0 checks, 23 lines checked
##[error]Bash exited with code '1'.

Earlier I was doing the indentation that way and got these errors.

Reply:
Use one tab